src/test/java/uk/gov/di/ipv/lambda/CredentialIssuerHandlerTest.java

```diff
@@ -123,6 +138,24 @@ void shouldReceive400ResponseCodeIfCredentialIssuerServiceThrowsException() thro
         verifyNoInteractions(context);
     }
 
+    @Test
+    public void shouldReturn500IfCredentialIssuerServiceGetCredentialThrows() throws JsonProcessingException {
```

Comment:
Can we remove the public here. Sonar will recognise it as a code smell and says that they should have default package visibility